frontend/lib/ifixit-api/devices.ts

```diff
@@ -16,6 +20,15 @@ export async function fetchDeviceWiki(
       );
       return await client.get(`cart/part_collections/devices/${deviceHandle}`);
    } catch (error: any) {
+      Sentry.withScope((scope) => {
+         scope.setExtra('headers', context?.req.headers);
+         scope.setExtra('url', context?.req.url);
+         scope.setExtra('method', context?.req.method);
+         scope.setExtra('device', deviceTitle);
+         scope.setExtra('local', context?.locale);
+         scope.setExtra('query', context?.query);
```

Comment:
I'd think about moving these to a context. It looks like `setExtra` is deprecated: https://docs.sentry.io/platforms/javascript/enriching-events/context/#additional-data